packages/web-components/fast-foundation/src/anchor/anchor.pw.spec.ts

```diff
@@ -0,0 +1,423 @@
+import { expect, test } from "@playwright/test";
+import type { FASTAnchor } from "./anchor.js";
+import { AnchorTarget } from "./anchor.options.js";
+
+test.describe("Anchor", () => {
+    const fixtureUrl = `http://localhost:6006/iframe.html?args=&id=anchor--anchor&viewMode=story`;
```

Comment:
I wonder if we should have a little test helper to generate the fixture url so we don't have to duplicate this in all the tests. Thoughts?

Reply:
Groovy.